README.md

```diff
@@ -1 +1,56 @@
-# math_magicians
\ No newline at end of file
+A todo-list project built with Webpack and ES6 modules
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
- [ ] Kindly use the latest readme file. You can find it [here](https://github.com/microverseinc/readme-template)

Reply:
Okay thanks for the feedback, I will address that.